src/assets/stylesheets/audio.scss

```diff
@@ -1,10 +1,17 @@
 .audio-setup-panel {
   text-align: center;
   overflow-y: auto;
+  display: flex; 
+  flex-direction: column;
+  height: 100%;
   
+  &__enter-button-container {
+    flex: 1;
+    display: flex;
+    justify-content: center;
```

Comment:
I think all the grant and enter containers should have a bottom margin of 24px, right now the buttons are too flush with the bottom of the dialog imho